cron/internal/format/json.go

```diff
@@ -86,6 +86,7 @@ type jsonScorecardResultV2 struct {
 // AsJSON exports results as JSON for new detail format.
 func AsJSON(r *pkg.ScorecardResult, showDetails bool, logLevel log.Level, writer io.Writer) error {
 	encoder := json.NewEncoder(writer)
+	encoder.SetIndent("", "  ")
```

Comment:
> If I change the BQ will it be an issue?

As far as I know, yes. [BQ expects newline delimited JSON](https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-json) and refers to the [linked spec](https://jsonlines.org/).

"Each Line is a Valid JSON Value". I can try to confirm with a test upload, but I expect this to break the BQ transfer.

Reply:
The files in the API bucket don't get transferred as far as I can tell. Would a pretty-print argument (bool?) be acceptable?